README.md

```diff
@@ -0,0 +1,15 @@
+# Adoptadog Application  
```

Comment:
Nice use of markdown styling. More info at https://guides.github.com/features/mastering-markdown/